asia/china/guizhou/Intro.md

```diff
@@ -0,0 +1,9 @@
+# Guizhou
+
+### Country: China
+
+- Mountains
+
+More information about Guizhou in [wikivoyage](https://en.wikivoyage.org/wiki/Guizhou) and [wikipedia](https://en.wikipedia.org/wiki/Guizhou)
+
+More information about China [here](../Intro.md)
```

Comment:
Please, don't use links like "here" as they are not accessible. Read more [about why click here links are not good](https://web.archive.org/web/20230324121734/https://webaccess.berkeley.edu/ask-pecan/click-here).